node-build.sh

```diff
@@ -30,5 +49,9 @@ fi
 rm -rf "$DIST"
 
 npm install
+if [ $WITH_TESTING ]
+then
+    npm run test:prod
```

Comment:
If you do the build first then the test can re-use some of the dependencies

Reply:
Not with the current karma configs, lets convert to jest first

Reply:
Agreed. Our project needs to switch to jest before we should even mess with this container.

PS: you're working way too late as well. (and yes I know I am as well, I'm still fixing bugs in my OS stuff because I neglected it during the holidays... :smile: )